packages/dfw/src/dfwcomponents/WorkOrderCell/StatusIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { View } from "react-native";
+import {
+    Icon,
+    Typography,
+    useStyles,
+    Color,
+    IconName,
+    EDSStyleSheet,
+} from "@equinor/mad-components";
+
+type StatusConfig = {
+    icon: IconName;
+    label: string;
+    textColor: Color;
+    iconColor: Color;
+};
+
+type StatusIconProps = {
+    statusConfig: StatusConfig;
+    hideLabel?: boolean;
+};
```

Comment:
This can probably be simplified by removing the `hideLabel` prop, and making `label` optional. If `label` is undefined, don't display it.

You can probably also simplify this even further by using `StatusConfig` as props directly.


```suggestion
type StatusIconProps = {
    icon: IconName;
    label?: string;
    textColor: Color;
    iconColor: Color;
};
```